osu.Game/Beatmaps/Formats/LegacyBeatmapDecoder.cs

```diff
@@ -456,52 +456,52 @@ private void handleEvent(string line)
             }
         }
 
+        private const char comma = ',';
+
         private void handleTimingPoint(string line)
         {
-            string[] split = line.Split(',');
+            var span = line.AsSpan();
+            Span<Range> ranges = stackalloc Range[span.GetSplitCount(comma)];
+            span.Split(ranges, comma);
 
-            double time = getOffsetTime(Parsing.ParseDouble(split[0].Trim()));
+            double time = getOffsetTime(Parsing.ParseDouble(span.Slice(ranges[0]).Trim()));
```

Comment:
Stylish preference here: `span.Slice(ranges[x])` or `span[ranges[x]]`. Two brackets for different meaning is somehow less readable.